diff --git a/README b/README
index 5bf82ba..77abbf3 100644
--- a/README
+++ b/README
@@ -27,6 +27,7 @@ ConCont     - See http://www.dydns.za.net for details
 DslReports  - See http://www.dslreports.com for details
 Sitelutions - see http://www.sitelutions.com for details
 Loopia      - See http://www.loopia.se for details
+Linode      - see http://www.linode.com for details
 
 DDclient now supports many of cable/dsl broadband routers. 
 
